.github/workflows/ci-4.x.yml

```diff
@@ -12,10 +12,10 @@ jobs:
             profile: ''
           - os: ubuntu-latest
             jdk: 8
-            profile: '-PtestNativeTransport'
+            profile: '-DaddMacSupport -PtestNativeTransport'
```

Comment:
I don't think we need this as we only test on Linux and Windows for now

Reply:
Changes from workflows reverted